posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -173,14 +181,15 @@ def to_data(col_val, col_idx):
             else:
                 return col_val
 
-        results = [[to_data(c, i) for (i, c) in enumerate(r)] for r in response.results]
+        results_mapped = [[to_data(c, i) for (i, c) in enumerate(r)] for r in results]
 
         return WebStatsTableQueryResponse(
             columns=response.columns,
-            results=results,
+            results=results_mapped,
             timings=response.timings,
             types=response.types,
             hogql=response.hogql,
+            hasMore=self.paginator.has_more(),
```

Comment:
There is a helper to put these params. It also adds `limit` and `offset` which is useful in case the backend chose defaults when it got nothing from the request/query.

```suggestion
            **self.paginator.response_params()
```